packs/crossplane-1.16/values.yaml

```diff
@@ -0,0 +1,193 @@
+pack:
```

Comment:
I assume by default Palette provider is not enabled. If so, you need to set the default state of the palette provider preset to disabled. This can be done by adding comments in this values.yaml. An example for reference can be found [here](https://github.com/spectrocloud/pax/blob/f9f706aa25d4445284e211e2ccf32375ad1579c2/stable/loggging_and_monitoring/prometheus-operator-57.0.1/values.yaml#L4)

Reply:
I want it disabled by default.  That is by design

Reply:
@3pings  - The preset - `"palette-provider` enables the Spectrocloud Palette provider. Similarly, you need to provide an option to disable it if they want to revert back. You can update the presets file with the additional preset for disabling (same group). Then, as Boobalan mentioned, we need to set the default to disabled.